packages/wonder-blocks-banner/src/theme/thunderblocks.ts

```diff
@@ -0,0 +1,61 @@
+import {mergeTheme} from "@khanacademy/wonder-blocks-theming";
+import {
+    border,
+    font,
+    semanticColor,
+    sizing,
+} from "@khanacademy/wonder-blocks-tokens";
+import defaultTheme from "./default";
+
+export default mergeTheme(defaultTheme, {
+    root: {
+        border: {
+            radius: border.radius.radius_080,
+            width: {
+                inlineStart: border.width.thin,
+                inlineEnd: border.width.thin,
+                blockStart: border.width.thin,
+                blockEnd: border.width.thin,
+            },
+        },
+        color: {
+            border: {
+                info: semanticColor.feedback.info.subtle.border,
+                success: semanticColor.feedback.success.subtle.border,
+                warning: semanticColor.feedback.warning.subtle.border,
+                critical: semanticColor.feedback.critical.subtle.border,
+            },
+            foreground: {
+                info: semanticColor.feedback.info.subtle.text,
+                success: semanticColor.feedback.success.subtle.text,
+                warning: semanticColor.feedback.warning.subtle.text,
+                critical: semanticColor.feedback.critical.subtle.text,
+            },
+        },
+    },
+    icon: {
+        color: {
+            info: semanticColor.feedback.info.subtle.icon,
+            success: semanticColor.feedback.success.subtle.icon,
+            warning: semanticColor.feedback.warning.subtle.icon,
+            critical: semanticColor.feedback.critical.subtle.icon,
+        },
+    },
+    button: {
+        layout: {
+            // Set the margin to negative to offset the padding of the button
+            marginInline: `calc(${sizing.size_080} * -1)`,
```

Comment:
I tried importing the component tokens from the button so that this could stay in sync. I wasn't able to though since we don't export the component tokens. Do we want to be able to use other component tokens in other packages?

Reply:
I think it is fine to keep it this way for now. This could be a good case for starting semantic non-color tokens at some point, and move this instead somewhere inside the `wb-tokens` package that could be consumed here and in `wb-button/wb-icon-button`.